Documentation/AdministratorManual/Index.rst

```diff
@@ -39,22 +36,22 @@ Edit file :file:`.htaccess` (or your virtual host) to read:
 .. _admin-manual-nginx:
 
 Installation (Nginx)
---------------------
+====================
 
 Edit your `server` block to read:
 
 .. code-block:: nginx
 
    location / {
-       rewrite ^/fileadmin/(?!(_processed_/)) /index.php last;
+      rewrite ^/fileadmin/(?!(_processed_/)) /index.php last;
 
-       # snip
+      # snip
    }
 
 or, if that better fits your setup, like that:
 
 .. code-block:: nginx
 
    location ~ /fileadmin/(?!(_processed_/)) {
-       rewrite ^(.+)$ /index.php last;
```

Comment:
If we want to be consistent, official documentation indents by 2 :)

Reply:
Like above: In official manuals once the docteam has to change comething in a file it goes for "one level = three spaces".

Reply:
I was wrong above, we were in a Nginx config. Maybe you've seen I've actually taken into account most of your suggestions, with you as author of the commit as you deserve it. But here it's not about indenting the "documentation" but indenting the code itself and I stick on following best-practises for the underlying language ;-)

Reply:
That's fine - but really, feel free to use whatever makes sense from the PR in you opinion. I just wanted to use the easiest way I could think of to get you informed :-)